documentation/api/parameters.md

```diff
@@ -0,0 +1,167 @@
+Authorization: Bearer fffffffffffffffffffffffffffffffffffffffffff=
+
+{
+    "methods": [
+        {
+            "moduleName": "SampleWebApp.dll",
+            "typeName": "SampleWebApp.Controllers.HomeController",
+            "methodName": "Index"
+        },
+        {
+            "moduleName": "System.Private.CoreLib.dll",
+            "typeName": "System.String",
+            "methodName": "Concat"
+        }
+    ]
+}
+```
+
+### Sample Response
+
+```http
+HTTP/1.1 202 Accepted
+Content-Type: application/json
+Location: localhost:52323/operations/67f07e40-5cca-4709-9062-26302c484f18
+```
+
+### Sample Output (Target Application)
+
+```
+info: DotnetMonitor.ParameterCapture.UserCode[0]
+      => SpanId:e40e62cffe1cf1cb, TraceId:c76b911969aa8abcf335907e96c62b33, ParentId:0000000000000000 => ConnectionId:0HMT2D6L8GT2Q => RequestPath:/ RequestId:0HMT2D6L8GT2Q:00000003 => SampleWebApp.Controllers.HomeController.Index (SampleWebApp)
+      SampleWebApp.Controllers.HomeController.Index(
+        this: 'SampleWebApp.Controllers.HomeController',
+        number: 10)
+info: DotnetMonitor.ParameterCapture.SystemCode[0]
+      System.String.Concat(
+        str0: 'firstString',
+        str1: '.secondString')
+```
+
+## Supported Runtimes
+
+| Operating System | Runtime Version |
+|---|---|
+| Windows | .NET 7+ |
+| Linux | .NET 7+ |
+| MacOS | .NET 7+ |
+
+## Additional Requirements
+
+- The target application must use ASP.NET Core.
+- `dotnet-monitor` must be set to `Listen` mode, and the target application must start suspended. See [diagnostic port configuration](../configuration/diagnostic-port-configuration.md) for information on how to do this.
+- The target application must have [`ILogger`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger) available via [ASP.NET Core's dependency injection](https://learn.microsoft.com/aspnet/core/mvc/controllers/dependency-injection).
+- This feature relies on a hosting startup assembly. If the target application [disabled automatic loading](https://learn.microsoft.com/aspnet/core/fundamentals/host/platform-specific-configuration#disable-automatic-loading-of-hosting-startup-assemblies) of these, this feature will not be available.
+- This feature relies on a [ICorProfilerCallback](https://docs.microsoft.com/dotnet/framework/unmanaged-api/profiling/icorprofilercallback-interface) implementation. If the target application is already using an `ICorProfiler` that isn't notify-only, this feature will not be available.
+- If a target application is using .NET 7 then the `dotnet-monitor` startup hook must be configured. This is automatically done in .NET 8+.
+
+## Additional Notes
+
+### Unsupported Parameters
+
+Currently some types of parameters are unable to be captured. When a method contains one of these unsupported types, the parameter's value will be represented as `<unsupported>`. Other parameters in the method will still be captured so long as they are supported.
```

Comment:
Could we describe the list of types that are supported, at least the common or easily describable ones?